customers/views.py

```diff
@@ -63,19 +69,9 @@ def retrieve(customer_id=-1):
                 "dob": f"{selected_customer.dob}",
                 "date_created": f"{selected_customer.date_created}",
             }
-            return json.dumps(returning_data)
+            return json.dumps(returning_data), 200
     except Exception:
-        return "ERROR, Code: 404, invalid id number entered"
-
-
-@views.route("/list", methods=["GET"])
-def list():
-    return retrieve()
-
-
-@views.route("/detail/<int:customer_id>", methods=["GET"])
-def detail(customer_id):
-    return retrieve(customer_id)
+        return {"Message": "ERROR, invalid id number entered"}, 404
 
 
 @views.route("/update/<int:customer_id>", methods=["PATCH", "PUT"])
```

Comment:
again, this route should not have a path. I should be able to call `PATCH` or `PUT` at the path `/customers/1` (if the ID is 1)